pages/blog/asyncapi-ambassador-program.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "AsyncAPI Ambasador Program"
+date: 2023-03-10T06:00:00+01:00
+type: Communication
+tags:
+  - Ambassador Program
+cover: /img/posts/ambassador-program/cover.webp
+authors:
+  - name: Barbaño González
+    photo: /img/avatars/barbano.webp
+    link: https://www.linkedin.com/in/barbano-gonzalez-moreno
+excerpt: Information about AsyncAPI Ambassador Program.
+featured: true
+---
+
+AsyncAPI and its community have continued growing exponentially during the last months. As a feedback exercise, both the project and its community are getting stronger one with another. The community is the driving force that leads the initiative. Consequently, the bigger it gets, the more consistency and potential the initiative has. It’s amazing to see the huge amount of people who invest part of their time in disseminating and promoting the initiative through talks, articles, workshops, and program direction... 
+
+To follow that line, the AsyncAPI Ambassador Program has been thought of as one of the latest additions to the initiative to keep the loop going. 
```

Comment:
```suggestion
To follow that line, the AsyncAPI Ambassador Program is one of the latest addition to the initiative to keep the community going. 
```

Reply:
additions is plural here, fixed it in commit suggestion